frappe/public/js/frappe/views/calendar/calendar.js

```diff
@@ -263,6 +263,12 @@ frappe.views.Calendar = class Calendar {
 			defaultView: defaults.defaultView,
 			weekends: defaults.weekends,
 			nowIndicator: true,
+			buttonText: {
+				today: __("Today"),
+				month: __("Month"),
+				week: __("Week"),
+				day: __("Day")
+			},
```

Comment:
Where is this exactly used?

Reply:
@shariquerik 

Your question seems very confusing, this change happens in calendar.js, which is basically the calendar view.

The buttons that the changes translates to are highlighted in red in the previously attached image.

When creating the Fullcalendar instance, the values of the buttons (Today, Month, Week Day) were assigned, which is the method indicated to translate these buttons by the fullcalendar documentation

Reply:
Is this solving your problem? can you add screenshots of **Before** & **After**

Reply:
this is JUST translation... and this solve de problema, u can see the imagem showing in pt-BR... beforme my change this show in english only.